features/support/shared_contexts/simple_git_repository.rb

```diff
@@ -8,24 +8,21 @@
   let(:root) { tmp_path.join('simple_app') }
   let(:lib_path) { root.join('lib') }
   let(:class1_path) { lib_path.join('class1.rb') }
-  let(:class1_reopen_path) { lib_path.join('class1_reopen.rb') }
   let(:class2_path) { lib_path.join('class2.rb') }
-  let(:class2_eval_path) { lib_path.join('class2_eval.rb') }
-  let(:module1_path) { lib_path.join('module1.rb') }
-  let(:module2_path) { lib_path.join('module2.rb') }
   let(:model1_path) { root.join('models', 'model1.rb') }
-  let(:locales_path) { root.join('locales') }
-  let(:name_locale_path) { locales_path.join('name.yml') }
-  let(:value_locale_path) { locales_path.join('value.yml') }
-  let(:item_view_path) { root.join('views', '_item.html.erb') }
-  let(:schema_path) { root.join('db', 'schema.rb') }
   let(:spec_path) { root.join('spec') }
   let(:class1_spec_path) { spec_path.join('class1_spec.rb') }
-  let(:factories_path) { spec_path.join('factories') }
-  let(:model1_factory_path) { factories_path.join('model1s.rb') }
-  let(:model1_factory_modification_path) { factories_path.join('model1s_modification.rb') }
   let(:action_view_shared_context) { spec_path.join('support', 'shared_contexts', 'action_view.rb') }
   let(:git) { Git.init(root.to_s) }
+  let(:spec_helper) { File.join(root, 'spec/spec_helper.rb') }
+
+  # By default, generate map just with metadata
```

Comment:
`generate map with metadata only` sound a bit better I think. but it is a stylish note, you can ignore it if you like original variant more :)

Reply:
You won't believe it, but I wrote exactly that and changed to this :smile:
I'm fine with either, so I'll change it.